reference/ToDo/README.md

```diff
@@ -0,0 +1,50 @@
+<p align="center">
+  <img src="doc/assets/To_Do_App_Banner.png">
+</p>
+
+# Uno Platform "Uno To Do" reference app
+`Uno To Do` is a beautifully designed sample app for [Uno Platform](https://platform.uno/) using the latest [Material Design 3 system](https://m3.material.io/).
+
+The design template makes it easy to jump-start or learn Uno Platform-powered mobile, web, and desktop applications. The app provides common functions such as logging on, recording tasks, adding due dates, setting reminders, and more.
+
+In addition, the sample code utilizes [Uno.Extensions](https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/ExtensionsOverviewAndGettingStarted.html) and establishes the best practices for cross-platform application design and development considering multiple screen sizes, accessibility, enforcing brand guidelines, and more. 
+
+Finally, you can use the [Uno To Do Figma template design]((https://www.figma.com/community/file/1110780207444745729)) together with the [Uno Platform for Figma plugin](https://www.figma.com/community/plugin/1045528009520465828/Uno-Platform) LivePreview feature to experience the app before even coding it. Export design assets and get clean, responsive XAML true to the way the app was designed.
+
+https://user-images.githubusercontent.com/16295702/170085831-e8457ddd-888e-4aff-bb68-30ed136dd108.mp4
```

Comment:
@nickrandolph, @kazo0 the ReadMe file will need to be adjusted for the ToDo Sample app.
But note that the previous [Uno To Do Figma template design]((https://www.figma.com/community/file/1110780207444745729)) is currently unpublished because with the latest plugin updates the file would have to be rebuilt from our latest Figma main file @NVLudwig shared.

So what should we do at the moment for this sample for the related ReadMe file? Left aside for now Figma related details?

Reply:
yeh just remove the figma details for now